lib/ansible/modules/network/panos/panos_address.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Ansible module to manage PaloAltoNetworks Firewall
+# (c) 2016, techbizdev <techbizdev@paloaltonetworks.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: panos_address
+short_description: create address service object
+description:
+    - Create address service object of different types [IP Range, FQDN, or IP Netmask].
+author: "Luigi Mori (@jtschichold), Ken Celenza (@itdependsnetworks), Ivan Bojer (@ivanbojer)"
+version_added: "2.3"
+requirements:
+    - pan-python
+options:
+    ip_address:
+        description:
+            - IP address (or hostname) of PAN-OS device
+        required: true
+    password:
+        description:
+            - password for authentication
+        required: true
+    username:
+        description:
+            - username for authentication
+        required: false
+        default: "admin"
+    address:
+        description:
+            - IP address with or without mask, range, or fqdn
+        required: true
+        default: None
+    address_name:
+        description:
+            - name of the address
+        required: true
+        default: None
+    type:
+        description:
+            - ip-netmask, fqdn, ip-range
```

Comment:
The `description:` need to describe the *what*
`choices:` (which needs adding) lists the valid options.

See http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#documentation-block for more info.